<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurações</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Empresa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_NomeEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Nome Fantasia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_NomeFantasia"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Endereço:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_EnderecoEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Número:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_NumeroEnd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Bairro:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_Bairro"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>CEP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_Cep"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Cidade:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_CidadeEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>UF:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_EstadoEmpresa">
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Telefone:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_TelEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>CNPJ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_CnpjEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>E-mail:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledt_EmailEmpresa"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="Btn_LogoEmpresa">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Imagem Logo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Ledt_LogoEmpresa">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Financeiro</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Porcentagem de Lucro:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Ledt_LucroEmpresa"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Taxa Débito padrão:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Ledt_debito"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Taxa Crédito Padrão:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Ledt_credito"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Fiscal</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>492</width>
            <height>752</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QCheckBox" name="CheckBox_emitNf">
             <property name="text">
              <string>Emitir Notas Fiscais</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Se você já emitiu nota com outro programa vai precisar preencher o número da última Nota-Fiscal e Nota-Fiscal ao Consumidor emitida(Geralmente apenas em produção).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="title">
              <string>NNF</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_39">
                <item>
                 <widget class="QLabel" name="label_33">
                  <property name="text">
                   <string>Número da última NF-e(homologação):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledit_NNfHomologNFe"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_40">
                <item>
                 <widget class="QLabel" name="label_34">
                  <property name="text">
                   <string>Número da última NF-e(Produção):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledit_NNFProdNFe"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_34">
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>Número da última NFC-e (Homologação):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledit_NNfHomolog"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_33">
                <item>
                 <widget class="QLabel" name="label_28">
                  <property name="text">
                   <string>Número da última NFC-e(Produção):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledit_NNfProd"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBox_Opcao">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Simples Nacional</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBox_Tpamp">
             <item>
              <property name="text">
               <string>0 - Homologação</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1 - Produção</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>ID CSC (ID Token):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledit_Idcsc"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>CSC:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledit_Csc"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QPushButton" name="Btn_schemaPath">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Pasta Schemas</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Lbl_SchemaPath">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QPushButton" name="Btn_CertficadoAcPath">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Pasta CertificadosAC</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Lbl_CertificadoAcPath">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QPushButton" name="Btn_certificadoPath">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Certificado</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Lbl_CertificadoPath">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Senha Certificado:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledit_CertificadoSenha">
               <property name="echoMode">
                <enum>QLineEdit::EchoMode::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Código UF:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledt_CUf"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_25">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Código Município:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledt_CMun"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Incrição Estadual:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Ledt_IE"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Responsável Técnico</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_27">
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>CNPJ:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_CnpjRespTec"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_28">
                <item>
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>Nome:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_NomeRespTec"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_29">
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Email:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_EmailRespTec"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_30">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Fone:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_FoneRespTec"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_31">
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>ID CSRT:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_IdCsrtRespTec">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_32">
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>HASH CSRT:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Ledt_HashCsrtRespTec">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Produto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_35">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>NCM Padrão:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledit_NCMProd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_36">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>CSOSN Padrão:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledit_CSOSNProd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_37">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>PIS/COFINS Padrão:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledit_PISProd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_38">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>CEST Padrão:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LEdit_CESTProd"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="Btn_Cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Aplicar">
       <property name="text">
        <string>Aplicar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
